app/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,7 @@
     <string name="now_playing_title">Now Playing</string>
     <string name="classic_song">Classic Song</string>
     <string name="playlist">Playlist</string>
+    <string name="favorite">Favorite</string>
```

Comment:
The text is "Favorite" or "Liked Song"?

Reply:
done

Reply:
that is favorite, and for like song i already add it on my new commit